Netable/Example/Services/NetworkService.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  SimpleNetworkService.swift
+//  Example
+//
+//  Created by Amy Oulton on 2022-11-28.
+//  Copyright © 2022 Steamclock Software. All rights reserved.
+//
+
+import Combine
+import Foundation
+import Netable
+
+class SimpleNetworkService {
+    static var shared = SimpleNetworkService()
+
+    private let netable = Netable(baseURL: URL(string: "http://localhost:8080/")!)
+
+    func getVersion() {
+        Task {
+            do {
+                let version = try await netable.request(GetVersionRequest())
+                print(version.buildNumber)
```

Comment:
If we're going to leave in some print statements, it might be nice to have some formatting and context for them

Reply:
I opted to remove this because now that I've corrected the fallback resource, it actually hits the "catch" block, which I feel is more significant anyhow as the logging is very clear.